arcade/gl/context.py

```diff
@@ -1,12 +1,14 @@
 import logging
+import typing
```

Comment:
delete import typing

Reply:
It was actually used in this file, `typing.cast()`.  I did this because `cast` already refers to something else.  But I think I should remove that bit anyway.